pom.xml

```diff
@@ -106,7 +106,6 @@
         <commons.compress.version>1.21</commons.compress.version>
         <commons.codec.version>1.13</commons.codec.version>
         <lang3.version>3.3.1</lang3.version>
-        <guava.version>30.1.1-jre</guava.version>
```

Comment:
Unpinned the guava dependency. Now ksql uses the one from common which is 32.0.1.